frontend/src/components/requisitionForm/RequisitionItemCard.tsx

```diff
@@ -81,8 +88,9 @@ const RequisitionItemCard: React.FC<Props> = (props) => (
         <Form.Item
           name={[props.field.name, "unitPrice"]}
           fieldKey={[props.field.fieldKey, "unitPrice"]}
-          rules={[props.rules.requiredRule, props.rules.moneyRule]}
+          rules={[FORM_RULES.requiredRule, FORM_RULES.moneyRule]}
           label="Unit Price"
+          normalize={(value: any) => parseInt(value)}
         >
           <Input prefix="$" type="number" placeholder="99.99" />
```

Comment:
This field doesn't let you enter numbers with decimals

Reply:
Fixed